rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -0,0 +1,24 @@
+#include "stdafx.h"
+#include "Emu/Memory/vm_ptr.h"
+#include "Emu/Cell/ErrorCodes.h"
+
+#include "sys_game.h"
+
+LOG_CHANNEL(sys_game);
+
+error_code _sys_game_board_storage_read(vm::ptr<u8> buffer, u8 code)
+{
+	if (!buffer)
+	{
+		return CELL_EFAULT;
+	}
+	sys_game.trace("sys_game_board_storage_read(buffer=*0x%x, code=0x%02x)", buffer, code);
```

Comment:
move it to the top of the function. And add spme newlines inbeteeen it looks cramped.